### DISCUSSION TITLE
Env URL and basePath redundant Warning

### DISCUSSION THREAD

**Discussion by wintangz** [nextauthjs/next-auth#10429] [Help]
Hello, I am upgrading from V4 to V5 and faced this warning:
`[auth][warn][env-url-basepath-redundant].`

I read the doc (https://authjs.dev/reference/warnings/#env-url-basepath-redundant) and it says that 

> "AUTH_URL (or NEXTAUTH_URL) and authConfig.basePath are both declared. This is a configuration mistake - you should either remove the authConfig.basePath configuration, or remove the pathname of AUTH_URL (or NEXTAUTH_URL). Only one of them is needed."

However, I have no idea what is the "authConfig.basePath" and can't find it anywhere. I tried removing the AUTH_URL in my .env but this cause the error `UnknownAction` when I try to sign in.
I am not sure what the problem is. It would be greatly appreciated if someone could help.

**Comment by rsippl** [comment]
I can confirm this. I need to set AUTH_URL (being behind a reverse proxy), and although I never set basePath in my auth config, I keep getting that warning. I bet this is a bug, the check for basePath (in case AUTH_URL is set, probably [here](https://github.com/nextauthjs/next-auth/blob/bad3e44819ca484f19a00569ce5a2677059427cb/packages/core/src/lib/utils/env.ts#L67)) is likely done after basePath is set to some default value.
Other than filling the logs with garbage, this probably does no harm.

**Comment by fabpico** [comment]
I opened an issue for this: https://github.com/nextauthjs/next-auth/issues/11522

**Comment by k-taro56** [comment]
We fixed this in #11636.
